### PR TITLE
Single position page

### PR DESCRIPTION
**Pull request by awojciak**

### CODE REVIEW

**Review by wojciech-cichocki — CHANGES_REQUESTED** [review]
Sometimes after closing position, UI show that I have no position, but I still have a few. It will fix after refreshing the page.

Screenshot from 2021-12-03 22-04-21

**Comment by wojciech-cichocki** [comment]
unclaimed fees simulation:
Screenshot from 2021-12-13 10-32-04

**Comment by wojciech-cichocki** [comment]
Negative liquidity:
Screenshot from 2021-12-13 10-39-05

**Review by PawelKrawczyk96 — CHANGES_REQUESTED** [review]
other coments below

**Comment by PawelKrawczyk96** [comment]
image
This button should be more wider (on position details)

**Comment by PawelKrawczyk96** [comment]
image
this header is a little bit lower than plot component

**Comment by PawelKrawczyk96** [comment]
claim fee button have wrong color on hover state.

**Comment by PawelKrawczyk96** [comment]
image
this hover state is not consistent with figma.

File: src/containers/NewPositionWrapper/NewPositionWrapper.tsx [hunk 1]
```diff
@@ -114,27 +112,30 @@ export const NewPositionWrapper = () => {
         const lowerTick = Math.min(ticksData[left].index, ticksData[right].index)
         const upperTick = Math.max(ticksData[left].index, ticksData[right].index)
 
+        console.log('liquidity calc by:', tokenAddress.toString())
+        console.log('pool token x:', allPools[poolIndex].tokenX.toString())
```
[the remark this answers is not in the document]
Comment (awojciak) [reply src/containers/NewPositionWrapper/NewPositionWrapper.tsx:116]: I'm gonna leave these by now, Konrad is using them

**Pull request merged**